.rubocop.yml

```diff
@@ -0,0 +1,50 @@
+AllCops:
+  TargetRubyVersion: 3.3.0
+  NewCops: enable
+  Exclude:
+    - 'bin/**/*'
+    - 'vendor/**/*'
+    - 'tmp/**/*'
+    - 'coverage/**/*'
+
+Gemspec/RequiredRubyVersion:
+  Enabled: false
+
+Layout/LineLength:
+  Max: 120
+  Exclude:
+    - 'decanter.gemspec'
+
+Lint/UnderscorePrefixedVariableName:
+  Exclude:
+    - 'lib/decanter/core.rb'
```

Comment:
I did not want to mess with functionality in this PR, so I opted to exclude certain rubocop configs on `core.rb` that would require more invasive changes.